ecosystem/sep-0024.md

```diff
@@ -1391,8 +1391,6 @@ There is a small set of changes when upgrading from SEP-6 to SEP-24.
 1. Removed `external_extra` transaction property since this should all live in a human readable ` more_info_url`.
 1. Changed the response of the deposit and withdraw endpoints from 403 to 200 since this is the expected flow.
 1. `/transactions` and `/transaction` are now required endpoints.
-1. `Transaction` properties `more_info_url, amount_in, amount_out, amount_fee, and stellar_transaction_id` are now
-   non-optional.
```

Comment:
This `Changes from SEP-6` section isn't comprehensive anymore and its unlikely to be useful given most businesses implement one or the other rather than migrate from SEP-6 to SEP-24.

This is out of scope of the current PR but at some point we should clean up the spec so its more focused.